repos/Directory.Build.targets

```diff
@@ -78,11 +82,15 @@
     <Exec Command="$(PatchCommand) %(PatchesToApply.Identity)"
           WorkingDirectory="$(ProjectDirectory)"
           Condition="'@(PatchesToApply)' != ''" />
+
+    <WriteLinesToFile File="$(RepoCompletedSemaphorePath)ApplyPatches.complete" Lines="complete" Overwrite="true" />
```

Comment:
Why `Lines="complete"` rather than an empty file?

Reply:
I was under the impression that Lines was required.  I see now that it's not.  I'll update.